ciao-controller/command.go

```diff
@@ -86,6 +86,21 @@ func (c *controller) deleteInstance(instanceID string) error {
 	return nil
 }
 
+func (c *controller) assignFloatingIP(floatingIP payloads.FloatingIP) error {
+	// get node id.  If there is no node id we can't send a delete
+	i, err := c.ds.GetInstance(floatingIP.AssignFloatingIP.InstanceUUID)
```

Comment:
As mentioned before we should not assume there exists a instance behind a given floating IP. We can start with this method, but longer term we may need two types of floating IP's. The ones with an instance behind them and ones where they are for virtual service end points.